app/frontend/src/components/views/workspace-form/WorkSpaceForm.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import PageTitle from '../../layouts/base/PageTitle';
+import labeled from '../../UI/labeled/Labeled';
+import Input from '../../UI/input/Input';
+import Button, { StyleType } from '../../UI/button/Button';
+import { createProjectFx } from 'store/projects';
+import TeammateList from '../../UI/teammate-list/TeammateList';
+import { Link } from 'react-router-dom';
+import { UiComponentText } from 'styles/Mixins';
+
+/**
+ * WorkspaceForm component props model
+ */
+interface Props { }
+
+/**
+ * WorkspaceForm component
+ */
+const WorkspaceForm: React.FC<Props> = () => {
+  const [title, setTitle] = useState('');
+
+  const submit = async (): Promise<void> => {
+    await createProjectFx({
+      title,
+    });
+  };
+
+  return (
+    <div>
+      <PageTitle>Workspace settings</PageTitle>
+      <Wrapper>
+        <LabeledInput
+          label='Workspace'
+          placeholder='Workspace title'
+          id='title'
+          value={ title }
+          onChange={ e => setTitle(e) } />
+        <LabelWrapper>
+          <label>Team management</label>
+          <Description>Current users in your team</Description>
+        </LabelWrapper>
+        <TeammateList>
+          <StyledLink to=''>
+            <StyledButton icon='plus'>Add new team member</StyledButton>
```

Comment:
click on this button should display a teammate form. I believe api for this form is already implemented

<img width="411" alt="Screenshot 2022-02-16 at 18 21 05" src="https://user-images.githubusercontent.com/31101125/154296525-6bd272af-7693-4e0d-ba53-5ff4f63a8098.png">

Reply:
@TatianaFomina I need your help.. 
1. I implemented teammate form but don't know how to hide and show
2. Should i add Team management label which indicates if there is teammates above the picture at TeammateList? because i don't see there is someway to get if teammates are appeared or not
3. I want to use the enum type object that is in the types directory but whenever i want to try to use this one it throws compiler error that i can't import any modules outside of src but it seems you used Task object which is it types directory at Taskinfo.tsx. How?

Reply:
1. You can set variable (for example, `isFormDisplayed`) as true once 'Add new team member' button is clicked. And then use conditional rendering to display form. 
https://en.reactjs.org/docs/conditional-rendering.html
https://en.reactjs.org/docs/hooks-state.html

3. Do you mean `ContactType` enum?
such import works for me: 
```import ContactType from 'types/entities/contactType';```

Reply:
Big thank you for you genius! I'll fix them as soon as i get home And i think i can check the length of array some like this `teammates.length` no?

Reply:
Resolved 1, 2 but 3 it still gives me a same error as before so i just added default value into model.
And `TeammateList` now has to render its label conditionally i moved label into `TeammateList` the rendering is fine but there is a problem that the label isn't rendering as expected since it uses store to set the condition.
I couldn't find solution that can mock up the store :disappointed: